2019-10-25 23:22:09,339 - [service.py: 36] -INFO: waiting for connections...
2019-10-25 23:24:59,323 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:7373...
2019-10-25 23:25:00,328 - [service.py: 25] -INFO: b'4444'
2019-10-26 00:01:01,562 - [service.py: 36] -INFO: waiting for connections...
2019-10-26 00:02:26,446 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:9977...
2019-10-26 00:02:27,450 - [service.py: 25] -INFO: b'4444'
2019-10-26 00:02:28,528 - [service.py: 29] -INFO: Connection from 127.0.0.1:9977 closed.
2019-10-26 00:03:25,452 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:10029...
2019-10-26 00:03:26,459 - [service.py: 25] -INFO: b'4444'
2019-10-26 00:16:52,070 - [service.py: 29] -INFO: Connection from 127.0.0.1:10029 closed.
2019-10-26 00:17:39,291 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:10908...
2019-10-26 00:17:40,294 - [service.py: 25] -INFO: b'4444'
2019-10-26 00:24:10,092 - [service.py: 36] -INFO: waiting for connections...
2019-10-26 00:24:32,739 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:11344...
2019-10-26 00:24:33,747 - [service.py: 25] -INFO: b'4444'
